unity-renderer/Assets/DCLServices/Lambdas/LambdasService.cs

```diff
@@ -49,6 +49,18 @@ public class LambdasService : ILambdasService
             return SendRequestAsync<TResponse>(wr, cancellationToken, endPoint, transaction, urlEncodedParams);
         }
 
+        public UniTask<(TResponse response, bool success)> Get<TResponse>(string url,
+            int timeout = ILambdasService.DEFAULT_TIMEOUT,
+            int attemptsNumber = ILambdasService.DEFAULT_ATTEMPTS_NUMBER,
+            CancellationToken cancellationToken = default,
+            params (string paramName, string paramValue)[] urlEncodedParams)
+        {
+            var wr = webRequestController.Ref.Get(url, requestAttemps: attemptsNumber, timeout: timeout, disposeOnCompleted: false);
+            var transaction = urlTransactionMonitor.Ref.TrackWebRequest(wr, url, finishTransactionOnWebRequestFinish: false);
```

Comment:
Be careful with this because the second parameter of `TrackWebRequest` is used to be sent to Sentry. The rest of requests are sending a little part of the url there, and you are sending the whole url here.